doc/man3/SSL_want.pod

```diff
@@ -71,7 +71,8 @@ A call to L<SSL_get_error(3)> should return B<SSL_ERROR_WANT_X509_LOOKUP>.
 =item SSL_RETRY_VERIFY
 
 The operation did not complete because an application callback set by
-SSL_CTX_set_cert_verify_callback() has asked to be called again.
+SSL_CTX_set_cert_verify_callback() has asked to be called again  via
```

Comment:
nit: `again via` with 2 spaces

Reply:
Fixed.

Reply:
BTW, would be nice if `doc-nits` caught things like that.